css/style.css

```diff
@@ -264,6 +273,10 @@ button.button-inline:hover {
 	overflow: hidden;
 	display: flex;
 	min-height: 40px;
+	position: fixed;
```

Comment:
The idea behind that was to keep the card title always on top even when scrolling though larger task lists. But I guess I'll remove this for now and try to add this in a separate PR. I guess it will also require some javascript magic.